src/collective/ifttt/profiles/default/actions.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0"?>
 <object name="portal_actions" meta_type="Plone Actions Tool"
-   xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+   xmlns:i18n="collective.ifttt">
```

Comment:
This is wrong. This defines only XML namespace and should be what it was. Domain is defined with `i18n_domain="collective.ifttt"`